playbooks/byo/openshift-cluster/upgrades/v3_3/upgrade.yml

```diff
@@ -61,5 +61,7 @@
 - include: ../../../../common/openshift-cluster/upgrades/upgrade.yml
   vars:
     openshift_deployment_type: "{{ deployment_type }}"
+    master_config_hook: "v3_3/master_config_upgrade.yml"
+    node_config_hook: "v3_3/node_config_upgrade.yml"
```

Comment:
This is my "hooks" idea. Unfortunately the path is relative to the common directory where it actually gets evaluated, not byo.